eclair-rpc-test/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcClientTest.scala

```diff
@@ -113,6 +114,16 @@ class EclairRpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
 
   behavior of "RpcClient"
 
+  it should "be able to create an invoice" in {
+    for {
+      c <- clientF
+      invoice <- c.createInvoice("test", None, None, None, None)
+    } yield {
+      println(invoice)
```

Comment:
I would mixin the [`BitcoinSLogger`](https://github.com/bitcoin-s/bitcoin-s/blob/839d520206023fa1075a65e19392d9f9b72e3a62/core/src/main/scala/org/bitcoins/core/util/BitcoinSLogger.scala) trait and use `logger` rather than `println`

Reply:
You should be able to tinker with logging levels in the [`common-logback.xml`](https://github.com/bitcoin-s/bitcoin-s/blob/796492a6781ee285fdbae17f207aae0f432e42f0/core/src/test/resources/common-logback.xml) file